src/extension/index.ts

```diff
@@ -98,7 +103,27 @@ function activateDiagnostics(disposables: Disposable[], store: Store, baser: Uri
         // When the user opens a doc, VS Code commonly silently opens the associate `*.git`. We are not interested in these events.
         if (doc.fileName.endsWith('.git')) return;
 
-        const artifactUri = baser.translateLocalToArtifact(doc.uri.toString(true /* skipEncoding */));
+        const artifactUri = (() => {
+            // TODO: Recall why skipEncoding=true in the common case.
+            // TODO: Review for consistent usage of skipEncoding (for example in `provideTextDocumentContent`)
```

Comment:
Can we get issues to track these and note the issues in the comment?

Reply:
Done: #448